projects/ngrx.io/content/guide/store/reducers.md

```diff
@@ -20,29 +20,14 @@ and the associated reducer function.
 First, define some actions for interacting with a piece of state.
 
 <code-example header="scoreboard-page.actions.ts">
-import { Action } from '@ngrx/store';
+import { createAction } from '@ngrx/store';
 
-export enum ActionTypes {
-  IncrementHome = '[Scoreboard Page] Home Score',
-  IncrementAway = '[Scoreboard Page] Away Score',
-  Reset = '[Scoreboard Page] Score Reset',
-}
-
-export class IncrementHome implements Action {
-  readonly type = ActionTypes.IncrementHome;
-}
-
-export class IncrementAway implements Action {
-  readonly type = ActionTypes.IncrementAway;
-}
-
-export class Reset implements Action {
-  readonly type = ActionTypes.Reset;
-
-  constructor(public payload: { home: number; away: number }) {}
-}
+export const homeScore = createAction('[Scoreboard Page] Home Score');
```

Comment:
No `map` used here, is this intentional?

Reply:
Yes, no map or unions for simplicity